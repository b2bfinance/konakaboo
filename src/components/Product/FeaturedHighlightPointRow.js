import React from 'react';
import styled from 'styled-components';
import GradeOutline from '@material-ui/icons/GradeOutlined';
import Typography from '@material-ui/core/Typography';
import Grid from '@material-ui/core/Grid';

export const Wrapper = styled(Grid)`
  background-color: ${props => props.theme.productBorder};
  border-top: 2px solid ${props => props.theme.productBorder};
  padding: 8px 16px;
`;

export const HighlightIcon = styled.span`
  color: ${props => props.theme.productFeaturedIconColor};
  font-size: 1.25rem;
`;

export const Feature = styled.span`
  margin-left: 4px;
`;

export default ({ feature }) => (
  <Wrapper container alignItems="center">
    <Grid item>
      <HighlightIcon>
        <GradeOutline fontSize="inherit" />
      </HighlightIcon>
    </Grid>
    <Grid item>
      <Typography variant="subtitle1">
        <Feature>{feature}</Feature>
      </Typography>
    </Grid>
  </Wrapper>
);
