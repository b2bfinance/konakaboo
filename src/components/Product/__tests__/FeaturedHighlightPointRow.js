import React from 'react';
import { render } from 'enzyme';
import FeaturedHighlightPointRow from '../FeaturedHighlightPointRow';

describe('FeaturedHighlightPointRow', () => {
  test('renders correctly', () => {
    expect(render(<FeaturedHighlightPointRow />)).toMatchSnapshot();
  });
});
