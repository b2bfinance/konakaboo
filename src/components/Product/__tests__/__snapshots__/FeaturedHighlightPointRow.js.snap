// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FeaturedHighlightPointRow renders correctly 1`] = `
.c0 {
  border-top: 2px solid;
  padding: 8px 16px;
}

.c1 {
  font-size: 1.25rem;
  line-height: 0;
}

.c2 {
  margin-left: 4px;
}

<div
  class="MuiGrid-container-1 MuiGrid-align-items-xs-center-9 c0"
>
  <div
    class="MuiGrid-item-2"
  >
    <span
      class="c1"
    >
      <svg
        aria-hidden="true"
        class="MuiSvgIcon-root-98 MuiSvgIcon-fontSizeInherit-104"
        focusable="false"
        role="presentation"
        viewBox="0 0 24 24"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <g>
          <path
            d="M12 7.13l.97 2.29.47 1.11 1.2.1 2.47.21-1.88 1.63-.91.79.27 1.18.56 2.41-2.12-1.28-1.03-.64-1.03.62-2.12 1.28.56-2.41.27-1.18-.91-.79-1.88-1.63 2.47-.21 1.2-.1.47-1.11.97-2.27M12 2L9.19 8.63 2 9.24l5.46 4.73L5.82 21 12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2z"
          />
        </g>
      </svg>
    </span>
  </div>
  <div
    class="MuiGrid-item-2"
  >
    <h6
      class="MuiTypography-root-107 MuiTypography-subtitle1-125"
    >
      <span
        class="c2"
      />
    </h6>
  </div>
</div>
`;
